charmcraft/commands/store/registry.py

```diff
@@ -290,12 +290,12 @@ class LocalDockerdInterface:
     def __init__(self):
         self.session = requests_unixsocket.Session()
 
-    def get_image_info(self, digest: str) -> Union[dict, None]:
-        """Get the info for a specific image.
+    def get_image_info_from_id(self, image_id: str) -> Union[dict, None]:
+        """Get the info for a specific image using its id.
 
-        Returns None to flag that the requested digest was not found by any reason.
+        Returns None to flag that the requested id was not found by any reason.
```

Comment:
pedantic: `found by any reason` -> `found for any reason` 

The same sentence is in the next method's docstring too.

Reply:
thanks! that was a silly mistake